ui/lib/utils/appOptions.ts

```diff
@@ -0,0 +1,41 @@
+import * as auth from '@lib/utils/auth'
+import * as i18n from '@lib/utils/i18n'
+
+export interface AppOptions {
+  token: string | null
+  hideNav: boolean
```

Comment:
I think `headless` is a little confusing, so I change it to `hideNav` to make it more specific, or we have some better naming.